web/modules/custom/dpl_webform/dpl_webform.module

```diff
@@ -16,6 +16,11 @@ function dpl_webform_webform_create(Webform $webform): void {
   // Therefore we disable saving of results for all new webforms.
   $webform->setSetting('results_disabled', TRUE);
   $webform->setSetting('results_disabled_ignore', TRUE);
+
+  // We want to set a new "default_value" to the "confirmation_type" setting.
+  // If we do not set this, the editors will have to manually change it when
+  // creating a new webform.
+  $webform->setSetting('confirmation_type', 'message');
```

Comment:
So what happens if the administrator deliberately want to choose another `confirmation_type` when they create the webform? Or is that even bot a possibility because that setting is hidden?

Reply:
The 'administer advanced webform confirmation settings' is now given to the 'editor' and 'local_administrator' roles. This permission gives them access to configuring the `confirmation_type`. So they can just change it to something else than the default value we set. 

The reason we set the default value at all is, that the normal default value redirects the user to the specific webforms page and show the confirmation message there. But we want the default behaviour to be to just reload the current page and show the confirmation massage.